sdk/java-sdk-spring/src/test/java/kalix/spring/badwiring/workflow/Main.java

```diff
@@ -31,6 +31,6 @@ public class Main {
 
   public static void main(String[] args) {
     logger.info("Starting Kalix Application");
-    SpringApplication.run(kalix.spring.badwiring.view.Main.class, args);
+    SpringApplication.run(Main.class, args);
```

Comment:
This was a mistake that I just noticed and fixing now. 

However, I'm quite confused by this. I'm sure we had test that would effectively run this classes to verify if our wiring rules are correct. But I can't remember how we did that and where is this code.

Reply:
Oh, I found it. Nevermind. 
It's in KalixSpringApplicationSpec

But it's not covering Workflows. I will fix it and add some more tests to prove that we can wire config + context for all components and component clients for Actions and Workflows.

Reply:
I didn't add extra tests here. It's easier to test if something can't be wired because Spring fails, but if it can be wired, Spring will just start as normal and doing that inside a unit test is producing all sort of other errors. 

We should certainly build something to assert it, but it will require some more investigation.

Reply:
created an issue for this: https://github.com/lightbend/kalix-jvm-sdk/issues/1888